server/memphis_handler_schemas.go

```diff
@@ -151,7 +150,8 @@ func (sh SchemasHandler) CreateNewSchema(c *gin.Context) {
 	}
 	exist, _, err := IsSchemaExist(schemaName)
 	if err != nil {
-		c.AbortWithStatusJSON(configuration.SHOWABLE_ERROR_STATUS_CODE, gin.H{"message": "Schema with that name already exists"})
+		serv.Warnf(err.Error())
```

Comment:
This is unexpected error and not a logic one so it is an error log. Change to error and add a string mentioned the name of the function as happen in other error logs